### ISSUE TITLE
List of Catalog Names

### ISSUE THREAD

**Issue by cpaternostro** [asascience/ott-coops#361]
- [x] Rename: "Water Surface Height Above Reference Datum" to "Water Levels"
- [x] Rename: "Wind Speed of Gust" to "Wind Gust"
- [x] Rename "Wind From Direction" to "Wind Direction"
- [x] Rename "Sea Water Temperature" to "Water Temperature"
- [x] Rename "Sea Surface Height Amplitude Due to Equilibrium Ocean Tide" to "Tide Predictions"
- [x] Rename "Sea Water Velocity To Direction" to "Currents Direction"
- [x] Rename "Sea Water Speed" to "Currents Speed" or "Water Velocity"
- [x] Rename "Sea Water Electrical Conductivity" as "Salinity"
- [x] It looks like the Wave data has already been changed to Significant Wave Height

**Comment by jamesdoyle21** [comment]
This ticket can go well beyond just the names Chris mentions here. Jeff pointed out the inconsistency in the legend item names between the obs drawer and the validation tool. When we get the chance, we should sit down and think about how we can improve this across the board.

These plots show he same data with completely different legends:
![image](https://user-images.githubusercontent.com/82955395/155024496-47b3ae22-ed39-4d16-bf8c-3d3ef0cbbc97.png)

**Comment by jamesdoyle21** [comment]
@cpaternostro if there are more areas / data titles we can improve, let's post them here to keep them in one place.

**Comment by jamesdoyle21** [comment]
Look into OM1 for names

**Comment by jamesdoyle21** [comment]
To add to this issue. We should make the font smaller and shorten long parameter names so they fit on one line, like OM1 does

![image](https://user-images.githubusercontent.com/82955395/155768527-980c0618-6fcc-4b65-92c5-ec802e20c5d9.png)

**Comment by jamesdoyle21** [comment]
@nguyandy This issue resurfaced in our meeting today

**Comment by nguyandy** [comment]
@jamesdoyle21 Are you referring to the inconsistencies between validation tool and the obs drawer?

**Comment by jamesdoyle21** [comment]
Yes, both that and inconsistencies in the observation plot itself

ie:
![image](https://user-images.githubusercontent.com/82955395/175051372-70224017-6759-4eca-bc17-c1361dbfc245.png)

**Comment by jamesdoyle21** [comment]
We didn't come to a great conclusion here though. Seems like we need to talk to @cpaternostro more about this
